optimum/utils/__init__.py

```diff
@@ -37,6 +40,22 @@ def is_onnxruntime_available():
     return _onnxruntime_available
 
 
+def is_onnxruntime_strict_available():
```

Comment:
Why "strict"?

Reply:
It is true if and only if the pypi package `onnxruntime is installed` (not `onnxruntime-gpu`)

Reply:
I agree with Michael, "strict" sounds a bit weird but I don't have much better ideas either. Would `is_onnxruntime_cpu_available` make sense? Or can we still run things on the GPU with `onnxruntime` but without `onnxruntime-gpu`?